plugins/modules/source_control/git_config.py

```diff
@@ -243,10 +255,10 @@ def main():
     if not module.check_mode:
         if unset:
             args.insert(len(args) - 1, "--" + unset)
-            cmd = ' '.join(args)
+            cmd = args
         else:
             new_value_quoted = shlex_quote(new_value)
-            cmd = ' '.join(args + [new_value_quoted])
+            cmd = args + [new_value_quoted]
         try:  # try using extra parameter from ansible-base 2.10.4 onwards
             (rc, out, err) = module.run_command(cmd, cwd=dir, ignore_invalid_cwd=False)
```

Comment:
Do all supported versions of ansible accept a list as the first argument to `module.run_command()`?

Reply:
Yes. It should have been supported for a very, very long time now (should have even be in the 1.x range).

Reply:
I played code archeologist a bit, and dug out the commit which added `run_command` - and it already supported lists: https://github.com/ansible/ansible/commit/300531507b9a261ed1707b4b1da6dabac0112b88 The first release this commit was contained in was Ansible 1.0. So yeah, a very, very long time indeed :)